src/emulator/storage/cloudFunctions.ts

```diff
@@ -77,9 +84,9 @@ export class StorageCloudFunctions {
   private createLegacyEventRequestBody(
     action: StorageCloudFunctionAction,
     objectMetadataPayload: ObjectMetadataPayload
-  ): string {
+  ) {
```

Comment:
Should we type this function? Since we're already importing `CloudEvent` from `events/types.ts`, we can also import `LegacyEvent` and use that here.

Reply:
Great call.